charts/dapr/values.yaml

```diff
@@ -39,8 +39,24 @@ global:
     workloadCertTTL: 24h
     allowedClockSkew: 15m
     controlPlaneTrustDomain: "cluster.local"
+    # If set to true, a bound service account token will be mounted and used to
+    # authenticate to Sentry.
+    mountSentryVolume: true
```

Comment:
Please update the README too with all the possible values

Reply:
Done, please take a look. Ideally we should be generating that README.md from the values file.